src/wwwroot/template-editor/adapterService.js

```diff
@@ -69,6 +76,20 @@
       $window.location = "/";
     }
 
+    function showModalAndRedirect(url, errorDetail) {
+      var modalOptions = {
+        title: $translate.instant('errors.server_error'),
+        message: errorDetail ,
+        buttons: [{
+          label: $translate.instant('accept'),
```

Comment:
What do you think about being more explicit about the behavior in the button label? for example: `Redirect to template list`. 

Maybe it is not so simple, we should think about how to inject our own custom texts. So, if it is not so easy, we could do it later.

Reply:
I guess there is not an easy and cleanest way to do that. I asked in the $translate github repo for tips about it. If they answer me with a nice approach I will do that in another task. You are agree @andresmoschini  to left this for later until I have better information?.

Reply:
> I asked in the $translate github repo for tips about it

In that case, it is a good idea adding a link here

>  If they answer me with a nice approach I will do that in another task. You are agree @andresmoschini to left this for later until I have better information?.

As I said before, it is something that we will need, but for the moment "accept" is fine. So, do not worry about it to finish this task.

Reply:
It is only for future reference, do not apply it now. 

Googling for another thing I casually found this: http://embed.plnkr.co/v1QsiEx4E11Z7Ba3irOp/

It seems to be related to what we will need.